androidApp/src/main/java/dev/johnoreilly/confetti/ui/ConfettiScaffold.kt

```diff
@@ -60,10 +64,30 @@ fun ConfettiScaffold(
     content: @Composable (ScaffoldState) -> Unit,
 ) {
     val authentication = koinInject<Authentication>()
+    val wearSettingSync = koinInject<WearSettingsSync>()
     fun signOut() {
         authentication.signOut()
         onSignOut()
     }
+
+    val coroutineScope = rememberCoroutineScope()
+
+    val wearSettingNodeState =
+        wearSettingSync.wearNodes.collectAsStateWithLifecycle(emptyList()).value
+    val wearSettingsUIState = WearUiState(wearSettingNodeState)
+
+    fun installOnWear() {
+        wearSettingNodeState.filter { !it.isAppInstalled }.forEach {
+            wearSettingSync.installOnWearNode(it.id)
+        }
+    }
+
+    fun updateWearTheme() {
+        coroutineScope.launch {
+            wearSettingSync.updateWearTheme()
+        }
```

Comment:
Can you move this to `WearSettingsSync`, similar to `installOnWear`? I think the logic and scoping should be pretty similar

Reply:
Aha! Yes good idea!